### ISSUE TITLE
Time Picker - Indicate live regions for dynamically changing content - (2036570365)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5611]
### Module: 
 35 Time Picker 

### Violation: 
 Indicate live regions for dynamically changing content 

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) 

### Instance ID: 
 2036570365 

### Severity: 
 9 

### Description: 
 [Issue]
There are dynamically changing content areas such as hour, minute, second, and AM/PM that do not indicate live regions when activating the following controls.

- Increase hour
- Decrease hour
- Increase minute
- Decrease minute
- Increase AM/PM
- Increase second
- Decrease second

[User Impact]
Screen reader users will be unaware of important updates to this content when they occur.

[Code Reference] 
 ```
<span aria-label="Increase hour" class="button button--hour-up button--top-left" role="button" tabindex="-1">
  (...)
</span>
<span aria-label="Hour" aria-valuemax="23" aria-valuemin="1" aria-valuenow="0" class="input hour" role="spinbutton" tabindex="0">--</span>
<span aria-label="Decrease hour" class="button button--hour-down button--bottom-left" role="button" tabindex="-1">
  (...)
</span>
 ``` 

### Note: 
 [Suggestion]
Indicate live regions for dynamically changing content. Live regions can be created by adding a role attribute set to "log", "status", "alert", "progressbar", "marquee", or "timer" as appropriate. Alternatively, custom behavior can be created by using the aria-live, aria-atomic, and aria-relevant attributes. Text injected into this live region element will be announced by screen readers.

Importantly, the element with the ARIA live attributes must be available when the page loads. Otherwise, many screen readers will not detect updates to the element. Additionally, the element must be empty on page load unless an immediate screen reader announcement is desired.
 

### Media Type: 
 Live Regions 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Related fix to https://github.com/Esri/calcite-components/issues/5582, could be applied in a similar way

**Comment by geospatialem** [comment]
Testing needed to determine if still exists, and the recommended fix.
